scripts/tests/chiptest/__init__.py

```diff
@@ -27,6 +30,179 @@
     os.path.join(_DEFAULT_CHIP_ROOT, "src/app/tests/suites"))
 
 
+@dataclass(eq=True, frozen=True)
+class BlackListedTest:
+    yaml: str
+    reason: str
+
+
+INVALID_TESTS = {
+    "tests.yaml",  # certification/tests.yaml is not a real test
+    "PICS.yaml",  # certification/PICS.yaml is not a real test
+}
+
+
+def _LoadBlacklistJson(json_file_path: str) -> Iterator[BlackListedTest]:
+    with open(json_file_path, 'rt') as f:
+        data = json.load(f)
+        for c in data["collection"]:
+            for name in data[c]:
+                yield BlackListedTest(yaml="%s.yaml" % name, reason=json_file_path)
+
+
+def _GetBlackListedTests() -> Set[BlackListedTest]:
+    blacklist = set()
+
+    # Flaky tests? these seem to pass though
+    blacklist.add(BlackListedTest(yaml="DL_Schedules.yaml", reason="Flaky"))
+    blacklist.add(BlackListedTest(yaml="Test_TC_LVL_2_1.yaml", reason="Flaky"))
+
+    # Examples. Should these be blacklisted?
```

Comment:
They should be prevented from running. Although for CI we are supposed to be getting the list of tests from src/app/tests/suites/ciTests.json which I am assuming will happen in a follow up

Reply:
I added a comment for now. ciTests works, however I am concerned about orphaned tests: what if a yaml is neither in ciTests nor in manualTests (like these)? Why do we have them then?